lib/agent/https_ocsp_agent.js

```diff
@@ -2,53 +2,39 @@
  * Copyright (c) 2015-2019 Snowflake Computing Inc. All rights reserved.
  */
 
-var Util = require('../util');
-var HttpsAgent = require('https').Agent;
-var SocketUtil = require('./socket_util');
+const SocketUtil = require('./socket_util');
+const { Agent } = require('urllib');
+const { buildConnector } = require('undici');
+
+function createConnectFunction (connector, mock = undefined) {
+  return function connect ({ hostname, host, protocol, port, servername, localAddress, httpSocket }, callback) {
+    return connector({ hostname, host, protocol, port, servername, localAddress, httpSocket }, (err, socket) => {
```

Comment:
Why is it better to have error in the first position and the socket in the second? Shouldn't we swap them?

Reply:
it's a used library callback